db/app.js

```diff
@@ -1,17 +1,25 @@
 const express = require("express");
 const app = express();
 
-const { viewCategories, getMessage } = require("./controllers/app.controller");
+const {
+  viewCategories,
+  getMessage,
+  viewReviews,
+} = require("./controllers/app.controller");
 
 app.use(express.json());
 
 app.get("/api", getMessage);
 
+app.get("/not-a-route");
+
 app.use((err, req, res, next) => {
```

Comment:
This should go after the app.get() function invocations. Also try to handle custom errors and psql errors in two separate error handling middlewares.